addon/webextension/background/loadSelector.js

```diff
@@ -25,19 +25,19 @@ window.loadSelector = (function () {
     let lastPromise = Promise.resolve(null);
     scripts.forEach((script) => {
       lastPromise = lastPromise.then(() => {
-        return chrome.tabs.executeScript({
+        return browser.tabs.executeScript({
           file: script
         }, () => {
-          if (chrome.runtime.lastError
-              && chrome.runtime.lastError.message != "Script returned non-structured-clonable data") {
-              catcher.unhandled(chrome.runtime.lastError, {script: script});
-              console.error("Error loading script", script, ":", chrome.runtime.lastError);
+          if (browser.runtime.lastError
+              && browser.runtime.lastError.message != "Script returned non-structured-clonable data") {
+              catcher.unhandled(browser.runtime.lastError, {script: script});
+              console.error("Error loading script", script, ":", browser.runtime.lastError);
```

Comment:
Is there some reason you're sticking with the callback error check here?

Seems like you could do 

```js
return browser.tabs.executeScript({ file: script }).catch(err => {
  if (err.message != "Script returned non-structured-clonable data") {
    catcher.unhandled(err, {script: script});
    console.error(`Error loading script: ${err}`);
  }
});
```

The code comments say that order matters, otherwise I'd suggest using a `Promise.all` here, which would nicely auto-fail if any of the scripts doesn't load.